test/unit/ngsiv2/general/contextBrokerOAuthSecurityAccess-test.js

```diff
@@ -238,7 +238,7 @@ describe('Secured access to the Context Broker with OAuth2 provider', function()
                 .matchHeader('fiware-service', 'smartGondor')
                 .matchHeader('fiware-servicepath', 'electricity')
                 .matchHeader('Authorization', 'Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJ3cHdWclJ3')
-                .get('/v2/entities/light1/attrs?attrs=state,dimming')
+                .get('/v2/entities/light1/attrs?attrs=state,dimming&type=Light')
```

Comment:
```
.get('/v2/entities/light1/attrs?attrs=state,dimming&type=Light')
````

vs:

```
.get('/v2/entities/light1/attrs')
.query({attrs: 'state,dimimng', type: 'Ligth'})
```

I though only the second alternative worked, but it seems both do work.

Just raising it just in case you prefer to change to the second. If not, just mark comment this as NTC.

Reply:
Maybe the second approach is a bit more clear but I would not spend effort changing this! :)

Reply:
NTC